File: CineBox/CineBox.Model/ViewModels/News.cs
using System;
namespace CineBox.Model.ViewModels
{
    public partial class News
    {
        public int Id { get; set; }

        public int CinemaId { get; set; }

        public string Name { get; set; } = null!;

        public string Description { get; set; } = null!;

        public DateTime CreatedDate { get; set; }
    }
}

File: CineBox/UI/cinebox_mobile/lib/models/News/news.dart
import 'package:json_annotation/json_annotation.dart';

part 'news.g.dart';

@JsonSerializable()
class News {
  int? id;
  int? cinemaId;
  String? name;
  String? description;
  DateTime? createdDate;

  News();

  factory News.fromJson(Map<String, dynamic> json) => _$NewsFromJson(json);

  Map<String, dynamic> toJson() => _$NewsToJson(this);
}
